.github/renovate.json

```diff
@@ -1,9 +1,6 @@
 {
   "extends": ["config:base"],
-  "packageRules": [
-    { "matchPackagePatterns": ["@senecacdot/satellite"], "rangeStrategy": "bump" },
```

Comment:
We might still need this, I don't think moving it into the repo will change how we do updates.